docs/07-faq/020-supported-clouds.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Which clouds are supported by Wing?
+id: supported-clouds
+keywords: [faq, supported clouds, winglang, Wing programming language, Wing language, AWS, GCP, Azure]
+---
+
+Wing was built from the ground up to make it an ideal choice for building applications on any cloud.
+Since wing is in early stages of developemnt it only supports the three biggest public clouds, with different levels of maturity:
+* AWS - Fully supported. If you are able to compile your Wing code for the simulator, you shoud be able to compile it for AWS as well (with the Terraform provisioning engine).
+* GCP - Partial support. Not any Wing code that compiles for the simulator will compile for GCP. 
+* Azure - Partial support. Not any Wing code that compiles for the simulator will compile for GCP. 
+
+We are working hard on filling the support gap between the different clouds, and plan to add support for more clouds in the future.
```

Comment:
Instead of saying we're planning to support more clouds, let's just ask folks what they want
```suggestion
We are working hard on filling the support gap between the different clouds -- check out our [roadmap](https://docs.winglang.io/status#roadmap) for more details. If there are other clouds you would like supported, please let us know through our GitHub or Slack!
```